packages/amazon-sumerian-hosts-three/src/three.js/index.js

```diff
@@ -20,59 +20,59 @@ import anim from './animpack';
 
 export default {
   /**
-   * @see env
-   */
-  env,
-  /**
-   * @see Utils
+   * @see module:hosts/core.Utils
    */
   Utils,
   /**
-   * @see MathUtils
+   * @see module:hosts/core.MathUtils
    */
   MathUtils,
   /**
-   * @see Deferred
+   * @see module:hosts/core.Deferred
    */
   Deferred,
   /**
-   * @see three.js/Messenger
+   * @see module:hosts/core.LipsyncFeature
    */
-  Messenger,
+  LipsyncFeature,
   /**
-   * @see three.js/HostObject
+   * @see module:hosts/core.GestureFeature
    */
-  HostObject,
+  GestureFeature,
   /**
-   * @see @amazon-sumerian-hosts/core/index/LipsyncFeature
+   * @see module:hosts/core.DefaultVisemeMap
    */
-  LipsyncFeature,
+  DefaultVisemeMap,
   /**
-   * @see GestureFeature
+   * @see module:hosts/core.DefaultGestureWords
    */
-  GestureFeature,
+  DefaultGestureWords,
   /**
-   * @see three.js/PointOfInterestFeature
+   * @see PointOfInterestFeature
    */
   PointOfInterestFeature,
```

Comment:
Shouldn't this be `@see threejs/PointOfInterestFeature`?

Reply:
Yes. Added prefix.